specification/conformance/build_cr_tables.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python3
+import json
+from collections import deque
+import logging
+import argparse
+import sys
+
+def init_logger(level):
+    # Create a logger instance
+    logger = logging.getLogger(__name__)
+    if (logger.hasHandlers()):
+        logger.handlers.clear()
+    match level:
+        case 'DEBUG':
+            logger.setLevel(logging.DEBUG)
+        case 'INFO':
+            logger.setLevel(logging.INFO)
+        case 'WARNING':
+            logger.setLevel(logging.WARNING)
+        case 'ERROR':
+            logger.setLevel(logging.ERROR)
+        case 'CRITICAL':
+            logger.setLevel(logging.CRITICAL)
+        case _:
+            logger.setLevel(logging.INFO)
```

Comment:
```suggestion
    logger.setLevel(getattr(logging, level.upper(), logging.INFO))
```
Can be simplified as above using Python's built in conversion.